mate-base/mate-applets/mate-applets-1.26.1-r1.ebuild

```diff
@@ -31,7 +31,7 @@ COMMON_DEPEND="
 	>=sys-apps/dbus-1.10.0
 	x11-libs/gdk-pixbuf:2
 	>=x11-libs/gtk+-3.22:3
-	x11-libs/gtksourceview
+	x11-libs/gtksourceview:3.0
```

Comment:
This is missing a new revision (git mv is fine), which `pkgcheck scan --commits` should warn about.

Reply:
I didn't add a revision as it will work for existing users of MATE and only affect new users so I'm saving the unnecessary rebuild. unless there is reason I've overlooked here to why I should force the bump.

--commit didn't warn on this one btw.

Reply:
No, that's wrong. It's automagic on existing systems. See https://devmanual.gentoo.org/general-concepts/ebuild-revisions/index.html

`pkgcheck scan --commits` should warn on it as you're changing RDEPEND without a new revision.

Reply:
I'll check what is going wrong my end then for it not trigger but in the meantime change is incoming.

Reply:
Later on I'll chat with you about the kind of scenarios where it breaks on IRC as you'll find it interesting, but let's get the fix in for now